NewForce-Cohort-4/nutshell-frugal-badgers
# Tf event list

# Description
Event branch that can be accessed once logged in. This feature will allow you to save events by their date, name, and location. Once you save one you can add it to the list of your events. 
Fixes #5 (Event List)
## Type of change
Please delete options that are not relevant.
 expected)
# Testing Instructions
Once you download the branch, you should be able to use the local servers to load the webpage and log in. Once you log in you should see an Events section and a button to create a new event. When you click the new event button the form will appear to fill out. Once the form is filled out you can save it and add it to your event list which will then populate that user's current list of events they have saved.

# Checklist:
